plaidml/bridge/openvino/src/plaidml_plugin/plaidml_plugin.cpp

```diff
@@ -15,6 +15,10 @@
 #include "ie_plugin_config.hpp"
 #include "inference_engine.hpp"
 
+// TODO: The awkward include of a *.cpp is to get `ngraph_register_interpreter_backend`. See below where it's used for
+// more info.
+#include "ngraph/test/runtime/interpreter/int_backend.cpp"  // NOLINT[build/include]
```

Comment:
I could avoid this by just writing out the contents of the function I'm including here: `ngraph_register_interpreter_backend`, as this is a short function. This is needed for the single_layer_tests to run. However, it really seems like that should be happening over on the testing side, rather than us registering the nGraph interpreter as part of the PlaidML plugin. I can't seem to find a place it's happening though, even for the other plugin tests; maybe I'm missing something?